File: week12/week12-2.py
#week12-2.py 列出 n以下的全部質數
n = int(input('請輸入1個數'))
def isPrime(n):
  for i in range(2,n): #只能 1 和 n 本身整除
    if n%i==0:
      return False
  return True #如果都沒有失敗，就是成功

for i in range(2,n+1):
  if isPrime(i): print(i, end=' ')

File: week12/week12-2.cpp
///week12-2.cpp 列出 n以下的全部質數
#include <stdio.h>
int isPrime(int n)
{///質數不能整除
    for(int i=2;i<n;i++){
        if(n%i==0) return 0;///失敗
    }
    return 1;///成功!!!結尾不能有功，有毒!!
}

int main()
{
    printf("請輸入1個數");
    int n;
    scanf("%d",&n);
    for(int i=2;i<=n;i++){///列出
        if(isPrime(i))
            printf("%d",i);
    }
}
